mikeio1d/cross_sections/cross_section_collection.py

```diff
@@ -221,6 +221,11 @@ def __or__(self, other) -> CrossSectionCollection:
         else:
             super().__or__(other)
 
+    @property
+    def cross_section_data(self) -> CrossSectionData:
```

Comment:
I am thinking if we should do the same for Res1D, i.e., call it result_data. Or we could call here just data as well. But I think it makes sense to streamline.

Reply:
Yeah I agree. I'll add a property for ResultData in Res1D as `result_data`